src/components/button/Button.js

```diff
@@ -0,0 +1,12 @@
+function Button({description, consoleLog, disabled}) {
+    return (<>
+        <button onClick={() => console.log({consoleLog})}
+                disabled={disabled}>
```

Comment:
Verder zou je de button is disabled enkel willen aanroepen wanneer je button uitgeschakeld wil hebben. Nu roep je hem steeds aan in je App.js. wat het nut van je component onderuit haalt.